vnc_auto.html

```diff
@@ -156,7 +157,7 @@
 <body>
     <div id="top_bar">
         <div id="status">Loading</div>
-        <div id="sendCtrlAltDelButton">Send CtrlAltDel</div>
+        <div id="sendCtrlAltDelButton">Send Ctrl+Alt+Del</div>
```

Comment:
I would prefer minimum necessary changes in this file.

Reply:
Done